### DISCUSSION TITLE
`pnpm create <packageName>@latest` uses old cached version and `pnpm create <packageName>` downloads the latest version!

### DISCUSSION THREAD

**Discussion by babakfp** [pnpm/pnpm#8288] [General]
Hi

The latest version of my package on NPM is `0.5.0`.

When I run the below command, it uses the cached old version.

```
pnpm create <packageName>@latest // v0.3.0
```

But, when I run it without the `@latest` part, it downloads and uses the latest version!

```
pnpm create <packageName> // v0.5.0
```

This makes no sense! What is the point of `@latest` if it's not going to get the latest version?

**Comment by zkochan** [comment]
The first time you run `<packageName>@latest` it gets cached for 24 hours. If you don't want this behaviour, then change this setting: https://pnpm.io/npmrc#dlx-cache-max-age

**Reply by babakfp** [reply]
Why does it just not do the expected thing? This is a bad user experience. If I wanted the cached version, I could do it without `@latest`. I'm sure there is some stupid behavior with `pnpm create <packageName>` too, and that's why every package guide recommends using `pnpm create <packageName>@latest` instead. Why does the command without `@latest` gets the latest version? That doesn't make sense. Nothing makes sense here.
